pkg/kubelet/api/v1alpha1/runtime/api.pb.go

```diff
@@ -1337,7 +1336,8 @@ type LinuxContainerSecurityContext struct {
 	// SELinux context to be optionally applied.
 	SelinuxOptions *SELinuxOption `protobuf:"bytes,4,opt,name=selinux_options,json=selinuxOptions" json:"selinux_options,omitempty"`
 	// The user to run the the container process as. Can be either UID or user
-	// name.
+	// name. If it's user name, it MUST exist in the container image, or else
```

Comment:
If the latter, the user MUST exist in the container image (i.e. in the /etc/passwd inside the image), and be resolved there by the runtime; otherwise, the runtime MUST error.